### ISSUE TITLE
Roadmap for Remixicon

### ISSUE THREAD

**Issue by iamrishishah** [Remix-Design/remixicon#597]
Numerous designers utilizing Remix Icons for their active projects often relay requests when they encounter issues due to current limitations in the iconography. As we aim to deliver our designs promptly to developers for implementation, we are eager to receive new iconography updates. Currently, we're uncertain about the timing of these updates, and we're left waiting for the next Remix icon release without any knowledge of its anticipated date or target month. Therefore, we request a shared roadmap plan to provide at least a rough idea of when updates might be expected. While we don't need an exact date, a targeted month would greatly assist us.

Moreover, I believe that many organizations might be willing to pay a certain fee if their iconography needs were prioritized for the next release. This could potentially generate more profit for the project. I trust that this suggestion could serve as additional motivation for your team. We genuinely appreciate the excellent work you do in supporting designers.

**Comment by xiaochunjimmy** [comment]
Very good suggestion, but at this stage I am unable to publish a Roadmap. However, there are also some other short-term solutions:

- Generally speaking, I will prioritize designing icons with the `Icon Request` tag added to their requests;
- I will try to release at least one version every month;
- For requests that are already in progress, I will try to add an `in progress` label;
- In addition, I will work hard to improve the Design Guideline so that the community can contribute together. This would be more efficient than me designing alone;
- Finally, you are our old friend. If there are any urgent icon needs, please let me know specifically and if I have time available, I will consider them as a priority.

**Comment by iamrishishah** [comment]
I agree that design guidelines would be very helpful for other designers who want to contribute to this project. I'm not sure what kind of documents you're planning for the guidelines, but I know that many designers are not familiar with how to design icons. They may only be familiar with using icon libraries. I suggest that the guidelines include instructions or video tutorials on how to design icons in Figma or Ai. This would be helpful for the designers, and it would also help us to improve our release process.

I'm very excited about this project, and I would love to contribute. Of course, you're not alone in this journey. I'm sure there are many other people who are also eager to help.
